cli/server/server.go

```diff
@@ -65,6 +66,10 @@ func NewCommands() []cli.Command {
 			Name:  "diff, k",
 			Usage: "Use if DB is restore from diff and not full dump",
 		},
+		cli.BoolFlag{
+			Name:  "state, r",
+			Usage: "Import state roots instead of blocks",
```

Comment:
Can we import both at the same time? Seems like a valid use case.

Reply:
Fixed.